__tests__/app.test.js

```diff
@@ -194,3 +194,306 @@ describe("GET /api/articles/:article_id", () => {
+            author: "icellusedkars",
+            body: "Archaeologists have uncovered a gigantic statue from the dawn of humanity, and it has an uncanny resemblance to Mitch. Surely I am not the only person who can see this?!",
+            comment_count: "0",
+            created_at: "2020-04-17T01:08:00.000Z",
+            title: "Does Mitch predate civilisation?",
+            topic: "mitch",
+            votes: 0,
+          },
+
+          {
+            article_id: 11,
+            author: "icellusedkars",
+            body: "Having run out of ideas for articles, I am staring at the wall blankly, like a cat. Does this make me a cat?",
+            comment_count: "0",
+            created_at: "2020-01-15T22:21:00.000Z",
+            title: "Am I a cat?",
+            topic: "mitch",
+            votes: 0,
+          },
+
+          {
+            article_id: 7,
+            author: "icellusedkars",
+            body: "I was hungry.",
+            comment_count: "0",
+            created_at: "2020-01-07T14:08:00.000Z",
+            title: "Z",
+            topic: "mitch",
+            votes: 0,
+          },
+        ];
+        expect(articles).toEqual(articlesDesc);
+      });
+  });
+  test("status:404, topic not found", () => {
+    const topic = "crepuscular";
+    return request(app)
+      .get(`/api/articles?topic=${topic}`)
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toEqual("No topic found for crepuscular");
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("status:200, responds with an array of comments for the given `article_id`", () => {
+    const ARTICLE_ID = 6;
+    const commentsArr = [
+      {
+        author: "butter_bridge",
+        body: "This is a bad article name",
+        comment_id: 16,
+        created_at: "2020-10-11T15:23:00.000Z",
+        votes: 1,
+      },
+    ];
+    return request(app)
+      .get(`/api/articles/${ARTICLE_ID}/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual(commentsArr);
+      });
+  });
+  test("status:200, responds with an array of comments for the given `article_id`", () => {
+    const ARTICLE_ID = 9;
+    const commentsArr = [
+      {
+        author: "butter_bridge",
+        body: "Oh, I've got compassion running out of my nose, pal! I'm the Sultan of Sentiment!",
+        comment_id: 1,
+        created_at: "2020-04-06T12:17:00.000Z",
+        votes: 16,
+      },
+      {
+        author: "icellusedkars",
+        body: "The owls are not what they seem.",
+        comment_id: 17,
+        created_at: "2020-03-14T17:02:00.000Z",
+        votes: 20,
+      },
+    ];
+    return request(app)
+      .get(`/api/articles/${ARTICLE_ID}/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual(commentsArr);
+      });
+  });
+  test("status:404, no comments found", () => {
```

Comment:
If the comment exists, but there are no comments, we'd actually want to return a 200.

We want to return a 404 if the article isn't found altogether, so if you put in article_id 9999 for example